_P208_Nokia_LCD_5110.ino

```diff
@@ -132,7 +135,8 @@ boolean Plugin_208(byte function, struct EventStruct *event, String& string){
         PCONFIG(5)= getFormItemInt(F("plugin_208_charsize_line_3"));
         PCONFIG(6)= getFormItemInt(F("plugin_208_GPIO_CE"));
         PCONFIG(7)= getFormItemInt(F("plugin_208_GPIO_DC"));
-
+        PCONFIG(8)= isFormItemChecked(F("plugin_208_debuglog")) ? 1 : 0;

```

Comment:
PCONFIG(8) is not supported.
See the macro:
```c++
#ifndef PCONFIG
  # define PCONFIG(n) (Settings.TaskDevicePluginConfig[event->TaskIndex][(n)])
#endif // ifndef PCONFIG
```
The size of `TaskDevicePluginConfig[taskindex]` is 8, so by using `PCONFIG(8)`, you're overwriting the first PCONFIG of the next task, or get out of bounds when using this on the last task.

Reply:
That's a pity. So it not possible to place more than 8 plugin-specific form fields....
It looks like, I have to remove the checkbox....

Reply:
You can also use `PCONFIG_LONG(0)`, where instead of 8 (0..7) there are 4 separate settings (0..3). 

Though there are more efficient ways to store settings, this will probably suffice.

Reply:
The PCONFIG(8) is replaced by PCONFIG_LONG(0)